examples/twitter-compose-with-typeahead/src/utils/isValidTwitterUsername.ts

```diff
@@ -0,0 +1,8 @@
+export function isValidTwitterUsername(username: string) {
+  return (
+    username.startsWith('@') &&
+    username.length > 1 &&
```

Comment:
Twitter starts suggesting from the first character.

```suggestion
    username.length >= 1 &&
```

Reply:
The first character of the handle without the "@" so technically this starting from 2 characters.

Reply:
Right—I guess I got confused, or my network was too slow 👍